channels/sv.m3u

```diff
@@ -30,3 +30,7 @@ http://cdn.elsalvadordigital.com:1935/wowtv/wowtv/master.m3u8
 http://cdn.elsalvadordigital.com:1935/wowtv/wowtv/playlist.m3u8
 #EXTINF:-1 tvg-id="OrbitaTVCanal25.sv" tvg-name="Órbita TV Canal 25" tvg-country="SV" tvg-language="" tvg-logo="https://3.bp.blogspot.com/-T6GUy9Xq7dk/XHqhBG7oE1I/AAAAAAAAHcI/JthcJF6_nZ8VDmiy0QGGXaVKrsgDsBvogCLcBGAs/s1600/%25C3%2593rbita%2BTV%2BCanal%2B25%2Bhd.png" group-title="",Órbita TV Canal 25
 https://5dcabf026b188.streamlock.net/orbitatv/livestream/chunklist_w1915865624.m3u8
+#EXTINF:-1 tvg-logo="https://i.imgur.com/Iy5jUjV.png" group-title="El Salvador", Canal 10| HD 
+http://streamingcws20.com:1935/tves/tves.smil/chunklist_w1862063957_b978000_sleng.m3u8
```

Comment:
It's streaming 24/7, however the channel doesn't stream continuously. Add [Not 24/7] next to the name.